website/docs/r/cce_cluster_v3.html.md

```diff
@@ -86,6 +86,12 @@ All above argument parameters can be exported as attribute parameters along with
 
   * `status` -  Cluster status information.
 
+  * `internal` - The internal network address.
+
+  * `external` - The external network address.
+
+  * `external_otc` - The endpoint of the cluster to be accessed through API Gateway.
```

Comment:
Hello @niuzhenguo did you know what "_otc" stand for ?

Reply:
what name of this attrubute would you like to change? There's already `internal` and `external` endpoints.